app/models.py

```diff
@@ -23,10 +24,20 @@ def set_password(self, password: str):
     def check_password(self, password):
         return check_password_hash(self.password_hash, password)
 
+    def get_booked_masterclasses(self) -> List[Union['Masterclass', None]]:
+        """
+        Returns a list of masterclass objects which the user is an attendee of.
+        """
+        booked_masterclasses = []
+        for bm in self.booked_masterclasses:
```

Comment:
note you could do this in one line:
`return [bm for bm in self.booked_masterclasses]`

This way is good too and it doesn't need to be changed, just as something to note.

Reply:
Oo yeah I like the list comprehension! Have updated.